src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Circle extends Shape implements Figure {
+    private int radius;
+    private String color;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
```

Comment:
better to use @Override annotation to make code more informative

Reply:
but only circle has radius

Reply:
> better to use @override annotation to make code more informative

but only circle has radius

Reply:
that was concerning method "draw" you didin't have the annotation in prev commit